docker-compose-dev.yml

```diff
@@ -164,24 +163,24 @@ services:
     tmpfs:
       - /mail:mode=770,size=128M,uid=1000,gid=1000
 
-  dev_keycloak:
-    container_name: "dev_keycloak"
-    image: 'quay.io/keycloak/keycloak:21.0'
+  keycloak:
+    container_name: keycloak
+    image: keycloak/keycloak:25.0
     hostname: keycloak
+    command:
+      - 'start-dev'
```

Comment:
This compose files seems to use all double quotes. For consistency, can we convert all single quotes to double quotes?